config/desktop/focal/environments/i3-wm/config_base/packages

```diff
@@ -1,5 +1,6 @@
 anacron
 apport-gtk
+apt-xapian-index
```

Comment:
Probably for fast application search when opening application launcher?

Reply:
while I was trying to build the image it has a step where it builds the Xapian index, since the package was not originally there, it wasn't installed, therefore failing the build process, it was listed in other desktop config_bases as well, so reused to ensure image builds without any issues

Reply:
> Probably for fast application search when opening application launcher?

well yes, it generates an axi-cache, enabling a faster command search